browser/ui/views/frame/vertical_tab_strip_region_view.cc

```diff
@@ -524,9 +524,12 @@ class VerticalTabStripRegionView::HeaderView : public views::View {
       return;
     }
 
+    std::vector<views::View*> children_copy;
+    base::ranges::copy(children(), std::back_inserter(children_copy));
+
     // View::ReorderChildView() didn't work for us. So remove child views and
     // add them again.
-    base::ranges::for_each(children(),
+    base::ranges::for_each(children_copy,
```

Comment:
Crash came from touching `children_` while iterating?
~~Also, curious why we don't use `RemoveAllChildViewsWithoutDeleting()` here?~~
Ah, it's deprecated.

Reply:
Yup :smile: 

That's much nicer - updated!